### PR TITLE
Updated docstrings for arcs_kt_plan and arcs_kt_schema

### PR DESCRIPTION
**Pull request by alxrsngrtn**
Added an examples of use.

### CODE REVIEW

File: third_party/java/arcs/build_defs/internal/kotlin.bzl [hunk 1]
```diff
@@ -401,7 +401,24 @@ def arcs_kt_android_test_suite(name, manifest, package, srcs = None, tags = [],
...
 def arcs_kt_plan(name, srcs = [], deps = [], visibility = None):
-    """Converts recipes in manifests into Kotlin Plans.
+    """Converts recipes from manifests into Kotlin plans.
+
+    Example:
+
+      Direct dependency on this target is required for use.
+
+      ```
+          arcs_kt_plans(
```
Comment (piotrswigon) [inline third_party/java/arcs/build_defs/internal/kotlin.bzl:411]: typo arcs_kt_plan

File: third_party/java/arcs/build_defs/internal/kotlin.bzl [hunk 2]
```diff
@@ -401,7 +401,24 @@ def arcs_kt_android_test_suite(name, manifest, package, srcs = None, tags = [],
...
+            name = "foo_plans",
+            srcs = ["foo.arcs"],
```
Comment (piotrswigon) [inline third_party/java/arcs/build_defs/internal/kotlin.bzl:413]: should we note a likely dependency on arcs_kt_schema output?

Comment (alxrsngrtn) [reply third_party/java/arcs/build_defs/internal/kotlin.bzl:413]: That's a good idea!

File: third_party/java/arcs/build_defs/internal/kotlin.bzl [hunk 3]
```diff
@@ -401,7 +401,31 @@ def arcs_kt_android_test_suite(name, manifest, package, srcs = None, tags = [],
...
+
+          arcs_kt_plan(
+            name = "foo_plans",
+            srcs = ["foo.arcs"],
+            deps = [":foo_schemas"],
+          )
+
+          arcs_kt_library(
+            name = "arcs_lib",
+            srcs = [glob("*.kt")],
```
Comment (csilvestrini) [inline third_party/java/arcs/build_defs/internal/kotlin.bzl:425]: Nit: glob returns a list, so this is just: `srcs = glob("*.kt")`

File: third_party/java/arcs/build_defs/internal/kotlin.bzl [hunk 4]
```diff
@@ -401,7 +401,31 @@ def arcs_kt_android_test_suite(name, manifest, package, srcs = None, tags = [],
...
+
+      Direct dependency on this target is required for use. This rule depends on the output from arcs_kt_schema.
+
+      ```
+          arcs_kt_schema(
+            name = "foo_schemas",
+            srcs = ["foo.arcs"],
+          )
+
+
```
Comment (csilvestrini) [inline third_party/java/arcs/build_defs/internal/kotlin.bzl:416]: Delete extra line

File: third_party/java/arcs/build_defs/internal/schemas.bzl [hunk 5]
```diff
@@ -58,13 +58,30 @@ def arcs_cc_schema(name, src, deps = [], out = None):
...
+
+      ```
+          arcs_kt_schema(
+            name = "foo_schemas",
+            srcs = ["foo.arcs"],
+          )
+
+          arcs_kt_library(
+            name = "arcs_lib",
+            srcs = [glob("*.kt")],
```
Comment (csilvestrini) [inline third_party/java/arcs/build_defs/internal/schemas.bzl:75]: Same here

**Pull request merged**